api/src/backend/tasks/jobs/attack_paths/prowler.py

```diff
@@ -0,0 +1,206 @@
+import neo4j
+
+from cartography.client.core.tx import run_write_query
+from cartography.config import Config as CartographyConfig
+from celery.utils.log import get_task_logger
+
+from api.db_utils import rls_transaction
+from api.models import Provider, ResourceFindingMapping
+from config.env import env
+from prowler.config import config as ProwlerConfig
+from tasks.jobs.attack_paths.providers import get_node_uid_field, get_root_node_label
+
+logger = get_task_logger(__name__)
+
+BATCH_SIZE = env.int("NEO4J_INSERT_BATCH_SIZE", 500)
+
+INDEX_STATEMENTS = [
```

Comment:
Should we add a filter to query by status? Apparently we use this filter for every query


```
"CREATE INDEX prowler_finding_status IF NOT EXISTS FOR (n:ProwlerFinding) ON (n.status);",
```